sklearn/metrics/classification.py

```diff
@@ -1517,11 +1547,16 @@ class 2       1.00      0.67      0.80         3
 
     report += u'\n'
 
-    # compute averages
+    # compute averages with specified averaging method
+    avg_p, avg_r, avg_f1, unused_s = \
+        precision_recall_fscore_support(y_true, y_pred, labels=labels,
```

Comment:
If we're going to do it this way (despite the fact that macro-averages can be calculated easily from the above class-wise scores), why wouldn't we allow multiple values to be provided to `average`? i.e. `average=['micro', 'weighted', 'macro']`